examples/get-pod1m-trip-data.py

```diff
@@ -0,0 +1,261 @@
+"""
+Script to get all Trip Data uploaded from an Arch Systems POD-1M device.
+
+The following describes the complete lifecycle of a POD-1M trip:
+
+1. When the POD-1M receives a command to start a trip (from the Companion App), the device creates a new entry on its
+   0e00 (Start Trip) stream. This represents the time at which the trip started.
+2. The POD-1M will start recording both environmental data (5021, 5022, and 5023) and shock events (5020).
+3. When the POD-1M receives a command to end the trip, the device creates a new entry on its 0e01 (End Trip) stream.
+   This represents the time at which the trip ended.
+4. All of this is so far on the device itself, but at some point after the trip is ended, the user will upload this data
+   to the cloud using the Companion App, at which time, all of these streams will be populated with data.
+5. The cloud is usually configured with a stream filter to send an email when the 0e01 signal is processed.
+   A filter could be changed to instead call a webhook on another server.
+6. After a device reviews the trip data on the cloud, the user is able to either reset the data (delete all the stream data/events)
+   or archive the trip, in which case all the data is kept but the streams are changed to the one for a Data Block which is equivalent
+   to a device (but read only). All the stream APIs work on either an active trip (device) or an archived trip (Data Block).
+   At this point, the device itself can be cleared and a new trip can be started.
+
+This script is intended to be used after step 5, which is when the trip was ended on the device and the data was uploaded to the cloud.
+
+Run like:
+
+     python3 -m venv  ~/.virtualenv/iotile-cloud
+     pip install iotile-cloud
+     python get_trip_data.py <device_slug>   # where device_slug can be `d--0abc` or `d--0000-0000-0000-0abc` just `0abc`
+
```

Comment:
I'm also not sure that this script provides an example of how the user can review/archive on the cloud, if that was intended, it just seems to collect.

Reply:
@mattrunchey This example is specific for how to get active data. It could be enhanced to handle archives, but I may rather write a second more script for that.